packages/rs-dpp/src/data_contract/document_type/class_methods/try_from_schema/v0/mod.rs

```diff
@@ -93,6 +94,7 @@ impl DocumentTypeV0 {
         default_keeps_history: bool,
         default_mutability: bool,
         validate: bool, // we don't need to validate if loaded from state
+        validation_operations: &mut Vec<ProtocolValidationOperation>,
```

Comment:
It says it's unused probably because of the valdiation feature flag. You probably need to mute this warning.

Reply:
Agreed just skip it

Reply:
Yep, it's fine for now